modules/publicize/publicize.php

```diff
@@ -743,6 +766,80 @@ function blog_id() {
 	 */
 	abstract function flag_post_for_publicize( $new_status, $old_status, $post );
 
+	/**
+	 * Ensures the Post internal post-type supports `publicize`
+	 *
+	 * This feature support flag is used by the REST API.
+	 */
+	function add_post_type_support() {
+		add_post_type_support( 'post', 'publicize' );
+	}
+
+	/**
+	 * Can the current user access Publicize Data.
+	 *
+	 * @param int $post_id. 0 for general access. Post_ID for specific access.
+	 * @return bool
+	 */
+	function current_user_can_access_publicize_data( $post_id = 0 ) {
+		/**
+		 * Filter what user capability is required to use the publicize form on the edit post page. Useful if publish post capability has been removed from role.
+		 *
+		 * @module publicize
+		 *
+		 * @since 4.1.0
+		 *
+		 * @param string $capability User capability needed to use publicize
+		 */
+		$capability = apply_filters( 'jetpack_publicize_capability', 'publish_posts' );
+
+		if ( 'publish_posts' == $capability && $post_id ) {
```

Comment:
Strict equality check here?

Reply:
Fixed in cc9d8e9f700906941fdc12c69185d153369f65cb.